apps/tailwind-components/components/input/Group.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="mb-4">
+    <div v-if="label">
+      <label :for="id" class="text-title font-bold">{{ label }}</label>
```

Comment:
The required label should be placed within the label. This ensures that required flag and the label properly describes the input element.

```html
<label>
    <span>{{ label }} </span>
    <span>Required </span>
</label>
```

Reply:
ok